src/cmd/sv-rollout/svrestarter.go

```diff
@@ -102,7 +107,7 @@ func (s *SvRestarter) notifyResult(result error) {
 	case ErrRestartPreempted:
 		s.log("was not required to restart in time", true)
 	default:
-		panic(result)
+		s.log(fmt.Sprintf("Unexpected error handled, likely a bug: %s : %s", reflect.TypeOf(result).String(), result), true)
```

Comment:
I think we should keep the panic so that we know the next time this happens - the logging is a good idea though.